docs/develop/go/test-suites.mdx

```diff
@@ -283,6 +286,102 @@ Learn to skip time manually in the SDK of your choice.
 
 Learn to skip time in Activities in the SDK of your choice.
 
+## Testing Nexus Operations {#test-nexus}
+
+A Nexus operation can be tested with a mock Nexus environment, which provides a way to mock the
+Nexus operation. This behavior allows you to test the Nexus operation in isolation by calling it
+directly without needing to create a Nexus handler service to run the Nexus operation.
```

Comment:
This isn't for testing a nexus operation, it's for testing a workflow that execute nexus operations.
Please clarify.